examples/rossmann-store-sales-preproc.ipynb

```diff
@@ -49,44 +55,35 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "INPUT_DATA_DIR = os.environ.get(\"INPUT_DATA_DIR\", \"/tmp/rossmann\")\n",
-    "OUTPUT_DATA_DIR = os.environ.get(\"OUTPUT_DATA_DIR\", \"./data\")"
+    "INPUT_DATA_DIR = os.environ.get('INPUT_DATA_DIR', './rossmann')\n",
+    "OUTPUT_DATA_DIR = os.environ.get('OUTPUT_DATA_DIR', './data')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The cell below will download the data into the `INPUT_DATA_DIR` if you do not have it already."
    ]
   },
   {
    "cell_type": "code",
    "execution_count": 4,
    "metadata": {},
-   "outputs": [
-    {
-     "name": "stdout",
-     "output_type": "stream",
-     "text": [
-      "--2020-08-27 16:32:48--  http://files.fast.ai/part2/lesson14/rossmann.tgz\n",
-      "Resolving files.fast.ai (files.fast.ai)... 104.26.2.19, 104.26.3.19, 172.67.69.159, ...\n",
-      "Connecting to files.fast.ai (files.fast.ai)|104.26.2.19|:80... connected.\n",
-      "HTTP request sent, awaiting response... 301 Moved Permanently\n",
-      "Location: https://files.fast.ai/part2/lesson14/rossmann.tgz [following]\n",
-      "--2020-08-27 16:32:48--  https://files.fast.ai/part2/lesson14/rossmann.tgz\n",
-      "Connecting to files.fast.ai (files.fast.ai)|104.26.2.19|:443... connected.\n",
-      "HTTP request sent, awaiting response... 200 OK\n",
-      "Length: 7730448 (7.4M) [application/x-gtar-compressed]\n",
-      "Saving to: ‘/input/rossmann.tgz’\n",
-      "\n",
-      "/input/rossmann.tgz 100%[===================>]   7.37M  37.9MB/s    in 0.2s    \n",
-      "\n",
-      "2020-08-27 16:32:48 (37.9 MB/s) - ‘/input/rossmann.tgz’ saved [7730448/7730448]\n",
-      "\n",
-      "googletrend.csv        state_names.csv\t test.csv\n",
-      "rossmann.tgz\t       store.csv\t train.csv\n",
-      "sample_submission.csv  store_states.csv  weather.csv\n"
-     ]
-    }
-   ],
+   "outputs": [],
+   "source": [
+    "isFile = os.path.isdir(INPUT_DATA_DIR)  \n",
+    "if isFile ==False:\n",
```

Comment:
things like ```if isFile == False``` should be ```if not isFile```.

I think here though we should check the existence of the file, rather than the existence of the INPUT_DATA_DIR
```suggestion
    "if not os.path.isfile(os.path.join(INPUT_DATA_DIR, "rossmann.tgz")):\n",
```